src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
```

Comment:
Change this to const App = () => ...

Reply:
i can actually remove the file since it is not used, it's the default file create-react-app creates